src/services/wyreService.js

```diff
@@ -188,6 +189,19 @@ class WyreService {
       };
     }
   }
+  getRates = async () => {
+    try {
+      const data = await this.service.get('https://api.sendwyre.com/v3/rates')
```

Comment:
we don't need to type the full url, see other examples in this file

Reply:
Fix according to above comment